setup.py

```diff
@@ -213,9 +211,9 @@ def run(self):
        'Intended Audience :: Developers',
        'License :: OSI Approved :: GNU General Public License v2 (GPLv2)',
        'Operating System :: POSIX :: Linux',
-       'Programming Language :: Python :: 2.7',
-       'Programming Language :: Python :: 3.4',
        'Programming Language :: Python :: 3.5',
```

Comment:
(although unrelated) Shouldn't we also test Python 3.5 if we officially support it?

Reply:
I think testing 3.6 and 3.7 is good enough for the upstream code base